packages/waku/src/lib/utils/form.ts

```diff
@@ -1,8 +1,16 @@
 // TODO is this correct? better to use a library?
-export const parseFormData = (body: string, contentType: string) => {
+export const parseFormData = (body: ArrayBuffer, contentType: string) => {
```

Comment:
Would you be able to create a test for this function?

Reply:
will try, going on vacation soon so have a lot of other projects to wrap up

Reply:
looking forward to it. take your time.